src/Pages/Logs.php

```diff
@@ -19,6 +20,14 @@ class Logs extends Page
 
     public ?string $logFile = null;
 
+    public static function canAccess(): bool
+    {
+        if (array_key_exists('view-logs' , Gate::abilities()) )
+            return Gate::allows('view-logs');
+
+        return true;
+    }
```

Comment:
Thanks for your contribution! Instead of using a hardcoded permission name (view-logs), it would be better to allow users to configure it. A good approach could be adding a configuration option in a config file.

For example:

```php
// config/filament-log-manager.php
return [
    .
    .
    .
    'permissions' => [
        'view_logs' => 'view-logs', 
    ],
];
```
Then, modify the method like this:

```suggestion
    public static function canAccess(): bool
    {
        $permission = config('filament-log-manager.permissions.view_logs');
        
        if ($permission && array_key_exists($permission, Gate::abilities())) {
            return Gate::allows($permission);
        }
        
        return true;
    }
```